drivers/windows/R/batch.R

```diff
@@ -80,10 +115,11 @@ path_bootstrap <- function(config) {
   version <- version_string(config$r_version, ".")
   if (platform == "windows") {
     ## TODO: update to I:/bootstrap(-dev)?/(windows|linux)/<version>
+    ## - Bit of a pain to migrate as I:/bootstrap is active.
+    ## - Can we tolerate I:/bootstrap, I:/bootstrap-dev  and
+    ##   /wpia-hn/Hipercow/bootstrap-linux /wpia-hn/Hipercow/bootstrap-dev-linux
```

Comment:
We'll need to fix this very soon, I think.  We can start with the new version saving things into -windows pretty soon. Can you make a ticket and we'll do this in the next few days after merging this

Reply:
https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-6247/Bootstrap-locations

Reply:
Provisionally gone for option 1 on that ticket - we'll talk tomorrow...